dbms/src/Common/SpaceSaving.h

```diff
@@ -255,6 +260,8 @@ class SpaceSaving
         writeVarUInt(size(), wb);
         for (auto counter : counter_list)
             counter->write(wb);
+
+        writeVarUInt(alpha_map.size(), wb);
```

Comment:
This explicitly stores the alpha map size in state in case, it should be the same most of the times though (unless someone actually stores this and then uses different K when finalizing state).